blackbelt/dependencies.py

```diff
@@ -66,8 +66,8 @@ def check(dep, list_path, licenses_path, dev=False, debug=False):
     """
     ensure_executables(['npm', 'license-checker'])
 
-    click.echo('Analyzing the package...')
     dep_name, dep_version = dep
+    click.echo('Analyzing the package '+dep_name+'@'+dep_version+' ...')
```

Comment:
Nitpick: whitespace around operators...

**Update:** Well, not. This should actually be:

```python
'Analyzing the package {}@{} ...'.format(dep_name, dep_version)
```

If you are interested in learning the quirks of the formatting, https://pyformat.info/ is a great resource. In Python 3.6+, you could also

```python
f'Analyzing the package {dep_name}@{dep_version} ...'
```

...which is cool, but unfortunately we did not drop Python 2 support on Black Belt yet 🙄